java-maven-sonar-argocd-helm-k8s/spring-boot-app/JenkinsFile

```diff
@@ -31,30 +32,29 @@ pipeline {
     }
     stage('Build and Push Docker Image') {
       environment {
-        DOCKER_IMAGE = "abhishekf5/ultimate-cicd:${BUILD_NUMBER}"
+        DOCKER_IMAGE = "srishtiv424/ultimate-cicd:${BUILD_NUMBER}"
         // DOCKERFILE_LOCATION = "java-maven-sonar-argocd-helm-k8s/spring-boot-app/Dockerfile"
         REGISTRY_CREDENTIALS = credentials('docker-cred')
       }
       steps {
-        script {
+        
             sh 'cd java-maven-sonar-argocd-helm-k8s/spring-boot-app && docker build -t ${DOCKER_IMAGE} .'
-            def dockerImage = docker.image("${DOCKER_IMAGE}")
-            docker.withRegistry('https://index.docker.io/v1/', "docker-cred") {
-                dockerImage.push()
-            }
-        }
+            sh 'docker login -u srishtiv424 -p Srishti123@ https://registry.hub.docker.com'
```

Comment:
You should not expose your credentials to all the people. You can store cred in jenkins or external tool and fetch it from there.